mitodl/open-discussions
# Refactor xpro course file sync

#### Pre-Flight checklist


#### What are the relevant tickets?
Closes #3740

#### What's this PR do?
Refactors the course file syncing for xpro to use the same function as for mitxonline - expecting individual course tarballs on S3 instead of a bundled tarball containing all courses.

#### How should this be manually tested?
- Set `XPRO_` and `MITX_ONLINE_` .env values to the same as on RC.  Ditto for `AWS_`.
- Run `backpopulate_xpro_data` followed by `backpopulate_xpro_files`.  Both should succeed.  
- Check that you have some `LearningResourceRun` objects and `Course` objects with `platform=xpro`, as well as numerous `ContentFile` objects with `run__platform=xpro`
- Run `backpopulate_mitxonline_data` followed by `backpopulate_mitxonline_files`.  Both should succeed.
- Check that you have some `LearningResourceRun` objectss and `Course` objects with `platform=mitxonline`, as well as numerous `ContentFile` objects with `run__platform=mitxonline`
